modules/commonFunction.js

```diff
@@ -0,0 +1,9 @@
+exports.generateRandomString = function(){
+	var text = "";
+	var possible = "123456789";
+
+	for (var i = 0; i < 4; i++) {
+		text += possible.charAt(Math.floor(Math.random() * possible.length));
```

Comment:
what this function do, give explanantion

Reply:
it genrates  random number  user id

Reply:
this function generates the random user id of four digit for users.

Reply:
no need of running the looop